stable/fluent-bit/values.yaml

```diff
@@ -5,8 +5,8 @@ on_minikube: false
 image:
   fluent_bit:
     repository: fluent/fluent-bit
-    tag: 1.0.6
-  pullPolicy: IfNotPresent
+    tag: 1.1.0
+  pullPolicy: Always
```

Comment:
Why change the pull policy? The image shouldn't ever change.

Reply:
I am reverting to the original value we had, Pull Policy was changed by 00a84538d7 in January this year without noticing it. 

From a maintainer perspective that's' our only metric to get a notion of the number of deployments. Users can still change to their preferred value.

Reply:
hmm. ok. Makes sense.

Reply:
@kfox1111 can you help to review the new property of Kube_Tag_Prefix ?

Reply:
hmm... so there is:
 Match               {{ .Values.filter.kubeTag }}.*
Kube_Tag_Prefix     {{ .Values.filter.kubeTag }}.

Both set to the same values, with one with a star and the other with just a dot. is that right? Do we need them to be separably set?

Reply:
kubeTag is set by default to 'kube'. but the docs for Kube_Tag_prefix show something like: kube.var.log.containers.

Maybe kubeTagPrefix should be its own value so its independently set able.

Reply:
actually Kube_Tag_Prefix by default must point to: 

```
{{ .Values.filter.kubeTag }}.var.log.containers.
```
here is an explanation of the changes and config to consider:

https://docs.fluentbit.io/manual/installation/upgrade_notes

Reply:
hmm make sense, I will do the change

Reply:
updated here: https://github.com/helm/charts/pull/13702/commits/2d7e21935516f8376c5f34dc2dd4ebc796f4c0c6

Reply:
@kfox1111 can you test it ?

Reply:
I don't have a quick place to do a test at the moment. sorry.